src/a4/domain/OpenSpace.java

```diff
@@ -1,5 +1,16 @@
 package a4.domain;
 
-public class OpenSpace {
+public class OpenSpace extends BoardSpace {
+	public OpenSpace() {
+	}
 
```

Comment:
I thought that Board Space should take a name, but we can just put it in Open and Property Spaces if that's all we're putting it in.